R/twas.R

```diff
@@ -158,6 +158,7 @@ harmonize_twas <- function(twas_weights_data, ld_meta_file_path, gwas_meta_file)
       for (study in names(gwas_files)) {
         gwas_file <- gwas_files[study]
         gwas_sumstats <- as.data.frame(tabix_region(gwas_file, query_region)) # extension for yml file for column name mapping
+        if(nrow(gwas_sumstats)==0) return (list(twas_data_qced = NULL, snp_info = NULL))
```

Comment:
This is nice. But I wonder if you can always add a message to it, like:

```
warning(paste("No GWAS summary statistics are found in the region of interest", query_region))
```

(you may want to format the region better)

Every time you return something like NULL it is best to include a message . Otherwise we may wonder what was causing the NULL